flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogManagerTest.java

```diff
@@ -460,10 +617,16 @@ void testCatalogStore() throws Exception {
                         Collections.emptyMap()),
                 ObjectIdentifier.of("exist_cat", "cat_db", "test_table"),
                 false);
+        catalogManager.createModel(
+                CatalogModel.of(Schema.derived(), Schema.derived(), Collections.emptyMap(), null),
```

Comment:
I'm fine using Schema.derived here but then you should not perform null checks in CatalogManager. Because Schema.derived is not null.